biosustain/GRASP
# Added sphinx docs and simulation_viz package along clean jupyter notebooks

All that Nick needs to do is:

  - run matlab's unit tests, the following will fail:
      'calculateExtremePathwaysTest/testCalculateExtremePathwaysRandom2' 
      'loadEnsembleStructureTest/testLoadEnsembleStructureWithFreeExchange'
      'plotControlAnalysisTest/testPlotControlAnalysisTest1'

  - run the unittests in visualization/simulation_viz/tests:
          - first create an environment using requirements.txt in visualization and 
          - then install simulation_viz manually

  - generate the sphinx docs: 
          - first create an environment using requirements.txt in docs folder
          - then run `make html`
          - open _build/index.html, take a look around and it should be good xD

 - make sure all cell in the jupyter notebooks (visualization folder) run
